src/webrtc/call.ts

```diff
@@ -292,6 +292,10 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
     public peerConn?: RTCPeerConnection;
     public toDeviceSeq = 0;
 
+    // whether this call should have push-to-talk semantics
+    // This should be set by the consumer on incoming & outgoing calls.
+    public isPtt = false;
```

Comment:
nitpick: would this be more legible long term as `isRadioMode` or similar?

Reply:
We've used 'ptt' in Element Call internally, so I think we'll probably stick with 'ptt' in the code for (hopefully) less confusion, irrespective of what it ends up being called to the user. And yep, I'm not yet 100% sure how we'll get the test coverage up on the voip code, but it's definitely a thing on my mind - thanks for the poke. :)